webpack/ForemanInventoryUpload/Components/InventoryFilter/__tests__/__snapshots__/integration.test.js.snap

```diff
@@ -14,6 +14,11 @@ Object {
   ],
   "state": Object {
     "ForemanRhCloud": Object {
+      "InsightsCloudSync": Object {
+        "settings": Object {
+          "insightsSyncEnabled": false,
+        },
+      },
```

Comment:
interesting, why was it updated?
it's under `InventoryFilter` directory.. not sure how it got here :)

Reply:
From the naming, I assume it actually loads all the components tree. So it loaded the whole InsightsSyncComponent too, since it appears under ForemanRhCloud.